src/addressHandler.ts

```diff
@@ -0,0 +1,187 @@
+import { db } from "./db/index.js";
+import { townTable } from "./db/town/schema.js";
+import { sql } from "drizzle-orm";
+
+export interface Address {
+  street: string; // alphabetic characters only, 1–3 words
+  number: number; // 1–999
+  floor: number; // 1–99
+  door: number; // 1–50
+}
+
+/** Including postal code & town name */
+export interface AddressWithTown extends Address {
+  postalCode: string;
+  town: string;
+}
+
+/** Town entity */
+export class Town {
+  #id: number;
+  #postalCode: string;
+  #name: string;
+
+  constructor(id: number, postalCode: string, name: string) {
+    this.#id = id;
+    this.#postalCode = postalCode;
+    this.#name = name;
+  }
+  getID() {
+    return this.#id;
+  }
+  getPostalCode() {
+    return this.#postalCode;
+  }
+  getName() {
+    return this.#name;
+  }
+  setPostalCode(postalCode: string) {
+    this.#postalCode = postalCode;
+  }
+  setName(name: string) {
+    this.#name = name;
+  }
+}
+
+/** Utility: random integer in [min, max] */
+export function randInt(min: number, max: number): number {
```

Comment:
Denne her funktion burde i virkeligheden rykkes til en util.ts fil, bruger selv en form for random i personHandler som kunne bruge denne her funktion